README.md

```diff
@@ -176,7 +174,7 @@ Creates a lambda function that notifies Slack via the [incoming webhooks](https:
 ### Example
 ```
   module "slack-notification" {
-    source  = "github.com/skyscrapers/terraform-codedeploy//notify-slack"
+    source  = "github.com/skyscrapers/terraform-codedeploy/notify-slack/functions"
```

Comment:
The module source hasn't changed, no?

Reply:
Ah, yes, that's me correcting a typo and then making a mistake. Let me fix that.